vernac/vernacexpr.ml

```diff
@@ -363,9 +362,13 @@ type nonrec vernac_expr =
   | VernacSolveExistential of int * constr_expr
 
   (* Auxiliary file and library management *)
-  | VernacAddLoadPath of rec_flag * string * DirPath.t option
+  | VernacAddLoadPath of { implicit : bool
```

Comment:
Is `implicit` the right name here? I seem to recall it was used somewhere else in the codebase but was a bit misleading. What does it mean here?

Reply:
Sorry, I missed it above. So, shouldn't it be `recursive` instead?

Reply:
Ah no, I see what the patch is doing: previously `isrec` was passed to `implicit`. This patch distinguishes the two terminologies more clearly.